packages/babylon/src/index.js

```diff
@@ -91,3 +111,16 @@ function getParserClass(
   }
   return cls;
 }
+
```

Comment:
Could `hasModuleSyntax` be simplified?
Is there ever a `child.type` of `ImportDeclaration` and friends that don't have the accompanioning `importKind` and `exportKind` values?

Reply:
I believe when the `flow` parsing isn't enabled, they aren't set.